source/language/types.rst

```diff
@@ -487,15 +510,16 @@ single-Unicode-character identifier.
 
    .. table:: [tab:real-constants] Built-in real constants in OpenQASM3 of type ``float[64]``.
 
-      +-------------------------------+--------+--------------+---------------------+
-      | Constant                      | ASCII  | Unicode      | Approximate Base 10 |
-      +===============================+========+==============+=====================+
-      | :math:`\pi`                   | pi     | π            | 3.1415926535...     |
-      +-------------------------------+--------+--------------+---------------------+
-      | :math:`\tau = 2\pi`           | tau    | τ            | 6.283185...         |
-      +-------------------------------+--------+--------------+---------------------+
-      | Euler’s number :math:`e`      | euler  | ℇ            | 2.7182818284...     |
-      +-------------------------------+--------+--------------+---------------------+
+      +-------------------------------+--------+----------------------------------+---------------------+
+      | Constant                      | ASCII  | Unicode                          | Approximate Base 10 |
+      +===============================+========+==================================+=====================+
+      | :math:`\pi`                   | pi     | - µ U+00B5 MICRO SIGN            | 3.1415926535...     |
+      |                               |        | - μ U+03BC GREEK SMALL LETTER MU |                     |
```

Comment:
Those are some pretty unusual representations of pi.

Reply:
oops i'll fix